plasmapy/formulary/collisions.py

```diff
@@ -1104,8 +1107,9 @@ def fundamental_electron_collision_freq(
        revised." Naval Research Lab. Report NRL/PU/6790-16-614 (2016).
        https://www.nrl.navy.mil/ppd/content/nrl-plasma-formulary
 
-    .. [3] J.D. Callen, Fundamentals of Plasma Physics draft material,
-       Chapter 2, http://homepages.cae.wisc.edu/~callen/chap2.pdf
+    .. [3] `Draft Material for "Fundamentals of Plasma Physics" Book
+       <https://docs.google.com/document/d/e/2PACX-1vQmvQ_b8p0P2cYsWGMQYVd92OBLX9Sm6XGiCMRBidoVSoJffj2MBvWiwpix46mqlq_HQvHD5ofpfrNF/pub>`__,
+       by James D. Callen
```

Comment:
This is a pretty fragile link, so I'm tempted to replace this a more persistent publication later on.  (It also gets repeated a bunch of times so I had to copy & paste a bunch, which gives me more motivation for a unified set of references as in #1245.)